### DISCUSSION TITLE
Do we a maven code to use/access appium2.0?

### DISCUSSION THREAD

**Discussion by PramodGD** [appium/appium#18268] [Q&A]
Hi Everyone,

We have automation framework which used appium 1.22.0 in maven dependency under java-client. Now we are upgrading to appium 2.0, Do we have a java-client version which supports appium 2.0? I use it under my POM.xml.

**Comment by jlipps** [comment]
Yes, [Java Client version 8](https://github.com/appium/java-client/blob/master/docs/v7-to-v8-migration-guide.md)
